test/rollback/rollbackAllPeer/rollbackAllPeer.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
I don't love doubling down on the bash script testing since we want to move towards using the nice docker-compose tooling we have (ie. #2805). Anyway we can use that here too?